pkg/pgmodel/sql_reader.go

```diff
@@ -119,137 +125,30 @@ func (q *pgxQuerier) Query(query *prompb.Query) ([]*prompb.TimeSeries, error) {
 	return results, err
 }
 
-func (q *pgxQuerier) LabelNames() ([]string, error) {
-	rows, err := q.conn.Query(context.Background(), getLabelNamesSQL)
-	if err != nil {
-		return nil, err
-	}
-
-	defer rows.Close()
-
-	labelNames := make([]string, 0)
-
-	for rows.Next() {
-		var labelName string
-		if err := rows.Scan(&labelName); err != nil {
-			return nil, err
+// fromLabelMatchers parses protobuf label matchers to Prometheus label matchers.
+func fromLabelMatchers(matchers []*prompb.LabelMatcher) ([]*labels.Matcher, error) {
```

Comment:
Can we rename this to `ProtoLabelMatcherstoLabelMatchers()`? and vice-versa for the other one? Makes it easier to understand, just like how its named in the upstream.

Reply:
I personally think that is too long and too verbose given how the function signature already shows that info. We can rename it if more people feel otherwise. I don't think we have the other version of the function, its just this one.